var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -466,7 +466,7 @@ def determine_version(cls, exe):
     @classmethod
     def determine_variants(cls, exes, version):
         def is_enabled(text):
-            if text in set(["t", "true", "enabled", "yes", "1"]):
+            if text.lower() in set(["t", "true", "enabled", "yes", "1", "on"]):
                 return True
             return False
```

Comment:
FWIW. You could simplify this further:

```suggestion
            return text.lower() in set(["t", "true", "enabled", "yes", "1", "on"])
```

Reply:
Good point:)